### PR TITLE
Verify no state leaks (e.g. connections) in TestServer

### PR DESCRIPTION
**Pull request by prashantv**
@akshayjshah

### CODE REVIEW

**Comment by akshayjshah** [comment]
lgtm. Can we also delete the `ServerState` method, since I don't think it has any callers now?

File: testutils/test_server.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```
Comment (akshayjshah) [inline testutils/test_server.go]: should be in a separate block.

**Comment by prashantv** [comment]
`ServerState` has already been removed.

File: testutils/test_server.go (hunk 1 above)
Comment (prashantv) [inline testutils/test_server.go]: :+1:

**Comment by akshayjshah** [comment]
lgtm.

**Pull request merged**